erpnext/controllers/queries.py

```diff
@@ -371,6 +371,19 @@ def get_account_list(doctype, txt, searchfield, start, page_len, filters):
 		fields = ["name", "parent_account"],
 		limit_start=start, limit_page_length=page_len, as_list=True)
 
+def get_blanket_orders(doctype, txt, searchfield, start, page_len, filters):
+	return frappe.db.sql("""select distinct bo.name, bo.blanket_order_type, bo.to_date
+		from `tabBlanket Order` bo, `tabBlanket Order Item` boi
+		where
+			boi.parent = bo.name
+			and boi.item_code = {item_code}
+			and bo.blanket_order_type = '{blanket_order_type}'
```

Comment:
```suggestion
			and bo.blanket_order_type = {blanket_order_type}
```

Reply:
@nabinhait  sql will process `blanket_order_type` as a column if there are no single quotes. I could either escape `blanket_order_type` (unnecessary) or  `repr(blanket_order_type)` before inserting into query. Or just let the single quotes be.